File: src/main/java/refactown/cleancode/c07exceptions/java/JsonException.java
package refactown.cleancode.c07exceptions.java;

public class JsonException extends RuntimeException {

	private static final long serialVersionUID = 1L;

	public JsonException(String message) {
		super(message);
	}

	public JsonException(String message, Throwable cause) {
		super(message, cause);
	}

}

File: src/main/kotlin/refactown/cleancode/c07exceptions/JsonException.kt
package refactown.cleancode.c07exceptions

import java.io.IOException

class JsonException : IOException{

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(message: String) : super(message)

    constructor(cause: Throwable) : super(cause)

    companion object {

        private val serialVersionUID = 1L
    }
}